### ISSUE TITLE
Rewrite whole package with TypeScript or create typing file

### ISSUE THREAD

**Issue by 5d-jh** [siyual-park/jwt-core#2]
This project doesn't provide typings, so it's little hard to use in real life.

Rewriting whole package with TypeScript or creating typing file would resolve this issue.

**Comment by siyual-park** [comment]
I agree with that opinion. But I don't have enough time to fix this. If only you are okay, I want to face this issue with you
